src/services/measure.service.js

```diff
@@ -143,6 +146,28 @@ const submitData = async (args, { req }) => {
   return output;
 };
 
+/**
+ * TO-DO: add bulk import functionality
+ * @param {*} args the args object passed in by the user
+ * @param {*} req the request object passed in by the user
+ */
+// eslint-disable-next-line no-unused-vars
+const bulkImport = async (args, { req }) => {
+  logger.info('Measure >>> $bulk-import');
+  throw new ServerError(null, {
+    statusCode: 400,
+    issue: [
+      {
+        severity: 'error',
+        code: 'BadRequest',
```

Comment:
I would change this to a 501 with NotImplemented response code since it is a valid request. Idk if any client will actually end up seeing this though...

Reply:
good point, I didn't know this was a response code